### PR TITLE
Change set_model_dir to set_model_from_file in demo files

### PR DESCRIPTION
**Pull request by gfwm2013**
test=develop

### CODE REVIEW

File: PaddleLite-armlinux-demo/image_classification_demo/image_classification_demo.cc [hunk 1]
```diff
@@ -236,7 +236,7 @@ int main(int argc, char **argv) {
 
   // Set MobileConfig
   paddle::lite_api::MobileConfig config;
-  config.set_model_dir(model_dir);
+  config.set_model_from_file(model_dir);
```
Comment (hong19860320) [inline PaddleLite-armlinux-demo/image_classification_demo/image_classification_demo.cc]: model_dir建议改成model_path

File: PaddleLite-armlinux-demo/object_detection_demo/object_detection_demo.cc [hunk 2]
```diff
@@ -245,7 +245,7 @@ int main(int argc, char **argv) {
   std::vector<std::string> word_labels = load_labels(label_path);
 
   paddle::lite_api::MobileConfig config;
-  config.set_model_dir(model_dir);
+  config.set_model_from_file(model_dir);
```
Comment (hong19860320) [inline PaddleLite-armlinux-demo/object_detection_demo/object_detection_demo.cc]: model_dir建议改成model_path

File: PaddleLite-armlinux-demo/image_classification_demo/image_classification_demo.cc (hunk 1 above)
Comment (gfwm2013) [reply PaddleLite-armlinux-demo/image_classification_demo/image_classification_demo.cc]: Done.

File: PaddleLite-armlinux-demo/object_detection_demo/object_detection_demo.cc (hunk 2 above)
Comment (gfwm2013) [reply PaddleLite-armlinux-demo/object_detection_demo/object_detection_demo.cc]: Done.

**Pull request merged**